src/provider/cloud-element-templates/util/propertyUtil.js

```diff
@@ -168,8 +168,11 @@ export function setPropertyValue(bpmnFactory, commandStack, element, property, v
     property
   };
 
+  let propertyHandled = false;
```

Comment:
What if we early return if handled instead? This pattern looks odd to me.

Reply:
We can't return early, same as we can't switch to `if else` branches. One property might have multiple handlers within the file, which can add more commands to the `commands` array.

We could return early for a simple `property` value, as this only has one handler, but this pattern would guarantee that it works for all future properties.

Reply:
Any other idea how we can re-structure this code path?

Reply:
My Proposal:
- Turn the paths into handler functions
- map property name to list of handlers
- if handler list is empty, throw error
- else: execute the commands, if any

I will try have this ready for the HoC tomorrow. Moving to "in progress" until then

Reply:
We decided to go with the NoOp solution to fix this bug, and to get rid of the `multi-command-executor` in the future, potentially during spring cleaning